todesk.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -ex
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-~/.var/app/com.todesk.ToDesk/data}
```

Comment:
This is set by Flatpak, you don't need to set it.

Reply:
Done.